considerations.mkd

```diff
@@ -8,8 +8,8 @@ As with other geographic data formats, e.g., [KMLv2.2], providing
 details about the locations of sensitive persons, animals, habitats, and
 facilities can expose them to unauthorized tracking or injury.  GeoJSON
 does not provide privacy or integrity services; if sensitive data
-requires privacy or integrity protection the service must be provided
-externally.
+requires privacy or integrity protection those must be provided by the
```

Comment:
@tschaub I like the suggestion of making a distinction between in-flight and at-rest. This is probably one of the few/first RFCs that people in the GIS business will read, and even though GeoJSON doesn't solve the problems, it could at least give GIS people the right vocabulary.

How about changing this to something like the following?

. If sensitive data requires privacy or integrity protection in flight, those must be provided by the transport, for example TLS or HTTPS. If privacy or integrity protection at rest is required, those must be provided by the storage system.